docs/develop/local_dev.mdx

```diff
@@ -63,50 +23,101 @@ import clsx from "clsx";
             BIN=/usr/local/bin && RESTATE_PLATFORM=x86_64-apple-darwin && \
             curl -LO https://github.com/restatedev/restate/releases/latest/download/restate.$RESTATE_PLATFORM.tar.gz && \
             tar -xvf restate.$RESTATE_PLATFORM.tar.gz && \
-            chmod +x restate restate-server && \
-            mv restate $BIN && \
-            mv restate-server $BIN
+            chmod +x restate restate-server
             ```
 
             ```shell MacOS-arm64
             BIN=/usr/local/bin && RESTATE_PLATFORM=aarch64-apple-darwin && \
             curl -LO https://github.com/restatedev/restate/releases/latest/download/restate.$RESTATE_PLATFORM.tar.gz && \
             tar -xvf restate.$RESTATE_PLATFORM.tar.gz && \
-            chmod +x restate restate-server && \
-            mv restate $BIN && \
-            mv restate-server $BIN
+            chmod +x restate restate-server
             ```
 
             ```shell Linux-x64
             BIN=/usr/local/bin && RESTATE_PLATFORM=x86_64-unknown-linux-musl && \
             curl -LO https://github.com/restatedev/restate/releases/latest/download/restate.$RESTATE_PLATFORM.tar.gz && \
             tar -xvf restate.$RESTATE_PLATFORM.tar.gz && \
-            chmod +x restate restate-server && \
-            mv restate $BIN && \
-            mv restate-server $BIN
+            chmod +x restate restate-server
             ```
 
             ```shell Linux-arm64
             BIN=/usr/local/bin && RESTATE_PLATFORM=aarch64-unknown-linux-musl && \
             curl -LO https://github.com/restatedev/restate/releases/latest/download/restate.$RESTATE_PLATFORM.tar.gz && \
             tar -xvf restate.$RESTATE_PLATFORM.tar.gz && \
-            chmod +x restate restate-server && \
-            mv restate $BIN && \
-            mv restate-server $BIN
+            chmod +x restate restate-server
             ```
         </CH.Code>
 
-        Then run it with:
+
+        Move the binaries to a directory in your PATH, for example `/usr/local/bin`:
+        ```shell
+        sudo mv restate $BIN && \
+        sudo mv restate-server $BIN
```

Comment:
I'm not a big fan of sudo commands... I think most distro these days have a `$HOME/.local/bin` where u can copy without being root. Perhaps check if you have it on your machine as well